crates/brainfuck_prover/src/components/memory/component.rs

```diff
@@ -0,0 +1,19 @@
+use stwo_prover::core::{channel::Channel, pcs::TreeVec};
+
+use super::table::N_COLS_MEMORY_TABLE;
+
+/// The claim for the Memory component
+#[derive(Debug, Eq, PartialEq)]
+pub struct Claim {
+    pub log_size: u32,
+}
+impl Claim {
+    pub fn log_sizes(&self) -> TreeVec<Vec<u32>> {
+        let trace_log_sizes = vec![self.log_size; N_COLS_MEMORY_TABLE];
+        TreeVec::new(vec![trace_log_sizes])
```

Comment:
```suggestion
            TreeVec::new(vec![vec![self.log_size; N_COLS_MEMORY_TABLE]])
```

Reply:
The TreeVec will have two other log size (for the preprocessed trace and the interaction trace (explained in the header doc pushed after another of your comments + a todo), it'll be much easier to read if the log_size vec! are created in their own variables imo